src/Steps/Table2.tsx

```diff
@@ -58,6 +70,25 @@ export const Table2 = () => {
     dispatch({ type: "MERGE_OPTION", payload });
   };
 
+  const options = [
```

Comment:
Did they have a specific order for this?  Should they be alphabetical or by the number at the end?